sklearn/_build_utils/__init__.py

```diff
@@ -15,6 +15,7 @@
 
 DEFAULT_ROOT = 'sklearn'
 # on conda, this is the latest for python 3.5
+# keep synced with pyproject.toml and circleci config
 CYTHON_MIN_VERSION = '0.28.5'
```

Comment:
Would it be ok to parse the pyproject.toml

Reply:
Likewise: https://github.com/scipy/scipy/blob/6fad1053e5aa14d5ecd5870517beb2c95a7481c7/tools/cythonize.py#L70